test/specs/WexExtManifestPlugin.spec.js

```diff
@@ -58,7 +58,7 @@ describe('WexExtManifestPlugin', () => {
     jest.doMock(
       path.resolve(options.manifest, 'common.manifest'),
       () => ({
-        mainfest_version: 2,
+        manifest_version: 3,
```

Comment:
There should be another `.spec.js` file for v3.

Reply:
yes sure, i'm going to have two different tests files